pages/operations/managing-applications/add-users-to-application.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Add Users to an Application
+description: How to give end users access to an application installed using Plural.
+---
+
+If you've enabled OIDC for a set of applications, you can easily give users or groups access to that application. To enable access to a particular application, you can add permissions via the Plural Marketplace or Plural Console. Any changes made in either the Marketplace or Console will be synced.
+
+## Through Plural Marketplace
+
+Admin users who have installed an application have the ability to add users to that application. To add users, navigate to the installed application in the Plural Marketplace. Click the application and select "OpenID Connect" in the sidebar on the left. This will open the form to add user or group bindings for this application. Add any new users or groups and click "Save" in the bottom right.
+
+![](</assets/operations/marketplace-add-users.png>)
+
+{% callout severity="info" %}
+When binding groups to an OIDC provider, this will not be reflected in the Console until a login from that group occurs.
```

Comment:
@michaeljguarino did I add this in the right place?

Reply:
I'm not sure we need the callout tbh, it's a very niche technical detail.  It's true of user and group bindings though.